problems/variables.js

```diff
@@ -54,6 +80,9 @@
 // * Output:
 // * 3
 
+// let a = 123 % 4;
```

Comment:
Although % 4 works for the number 123, it does not work for other numbers. Try using % 10 to get the last digit of any number.